### DISCUSSION TITLE
When is clicking on a link functionality within a snap going to be released?

### DISCUSSION THREAD

**Discussion by javaspeak** [MetaMask/snaps#1913] [Q&A] [answered]
When is clicking on a link functionality within a snap going to be released?

I heard that is being worked on already so there is no need to raise a SIP (request for new functionality).

Will / Is this link be shown a bit like?:

```
content: panel([
    heading('Transaction Trust Insights Snap'),
    text(
        `This to address has negative feedback: ${transaction.to}`,
    ),
    link( `click for more info', 'https://someurl.com?to=${transaction.to}` ),
    text(
        'Have a Great Day!',
    ),
]),
```

**Comment by FrederikBolding** [comment, marked as answer]
This feature should release within a couple of weeks. We shipped most of the functionality for this recently, but are still working through some last hurdles. 

You'll be able to use them via Markdown, e.g. `text("View on [Etherscan](https://etherscan.io)")`.

**Reply by javaspeak** [reply]
In the end was this released?

**Reply by Montoya** [reply]
Yes this is possible now

**Reply by javaspeak** [reply]
Does the link have to be https? 

Seems like that when I am in dev mode and use a http link like:   http://localhost:3000/blah that it errors and says timed out calling localhost:8080. Do not know why the port changed when it is not https.

I think the link functionality does NOT support http and only https - is that correct?  Cannot find this info in the current docs.

**Reply by Montoya** [reply]
Sorry for the late reply, it has to be https
